jin_liaskos/week4/wk4_d1/calculator.rb

```diff
@@ -0,0 +1,39 @@
+#calculator
+
+calculators = ['1. +', '2. -', '3. *', '4. /', '5. Quit']
+
+puts calculators
+
+puts "Choose the operation "
+
+choice = gets.chomp
+
+puts "Enter first number"
+num1 = gets.chomp.to_i
+
+puts "enter second number"
+num2 = gets.chomp.to_i
+
+case choice
+when "1" || "+"
```

Comment:
This `when "1" ||  "+"` syntax does not work. For `case..when` you have to use commas, so this is the correct format:
```ruby
when "1", "+"
```
The comma here behaves just like the `||`